build.gradle

```diff
@@ -29,4 +29,7 @@ dependencies {
     implementation('org.springframework.boot:spring-boot-starter-web')
     runtimeOnly('com.h2database:h2')
     testImplementation('org.springframework.boot:spring-boot-starter-test')
+    compile('mysql:mysql-connector-java')
```

Comment:
compile oraz implementation to ten sam scope, tylko inna nazwa. Warto trzymac sie nowego nazewnictwa
compile jest depricated 
https://stackoverflow.com/a/44493379